### ISSUE TITLE
A11y_SQLAzureDataStudio_SQLServerBigDataCluster_ConfigurationProfile_Screenreader: Repeatative announcement of radiobutton while navigating to each cell of the table while using NVDA 

### ISSUE THREAD

**Issue by shweta255** [microsoft/azuredatastudio#9342]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags:#A11yMAS;#A11yTCS;#SQL Azure Data Studio;#DesktopWeb;#Win32;#MAS1.3.1;#Benchmark;#A11y_SQLAzureDataStudioBenchmark;#A11ySev2;#NVDA;

**Environment Details:**
Version: 1.16.0-insider
Commit: 506c6a5
Date: 2020-02-17T05:43:18.399Z
VS Code: 1.42.0
Electron: 7.1.11
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.18363
Additional Details:
MAS References: MAS1.3.1
NVDA Version: 2019.2

**Repro Steps:**
1. Launch Azure Data Studio.
2. Select the Connections icon (if not already selected) from left bar.
3. Click on the ellipses '...' and select "New Deployment".
4. Select "SQL Server Big Data Cluster" card under deployment options.
5. Select the version & checkbox.
6. Navigate till Selet button & hit enter.
7. Launch NVDA screenereader
8. Navigate to table present under radiobutton using down arrow key

**Actual:**
While NVDA is on and navigated to tables of radiobutton using down arrow key, screenreader repeatatively announce radiobutton with checked/ uncheched status while navigating to each cell of the table

**Expected:**
While NVDA is on and navigated to tables of radiobutton using down arrow key, screenreader repeatatively should not announce radiobutton + it's checked/ uncheched status while navigating to each cell of the table

**User Impact:**
Screenreader users will face difficulty as repeatative radiobutton announcement is done by screenreader.

**Attachment link reference:**
[9342_A11y_SQLAzureDataStudio_SQLServerBigDataCluster_ConfigurationProfile_Screenreader.zip](https://github.com/microsoft/azuredatastudio/files/4274768/9342_A11y_SQLAzureDataStudio_SQLServerBigDataCluster_ConfigurationProfile_Screenreader.zip)

**Comment by alanrenmsft** [comment]
for this particular issue, seems to be a voice over issue, i can't really control how voiceOver read the information. but i do see a lot of problems with the card component design. we might need a redesign.

**Comment by BhanuPrakashGit** [comment]
@alanrenmsft Could you please the reopened the bug as it should be verified the Trusted Tested.
Please provide the latest environment details where we can verify this and take action accordingly.

GithubTags:#Regressed:06-02-23;

**Comment by BhanuPrakashGit** [comment]
As "SQL Server Big Data Cluster" has been removed.
**GithubTags:**#Closed;

**Comment by mstechie** [comment]
GitHubTags:#SQLADS-Win32-Feb2020;
